src/Microsoft.Diagnostics.Monitoring.WebApi/Stacks/JsonStacksFormatter.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi.Stacks
+{
+    internal sealed class JsonStacksFormatter : StacksFormatter
+    {
+        public JsonStacksFormatter(Stream outputStream) : base(outputStream)
+        {
+        }
+
+        public override async Task FormatStack(StackResult stackResult, CancellationToken token)
```

Comment:
Feel free to ignore - it feels slightly confusing to have two variables of type `StackResult`, one named `stackResult`, and one just named `result`. Can we better distinguish between the purposes/meanings of them?